design/ceph/object/store.md

```diff
@@ -712,3 +712,64 @@ For external CephCephObjectStores (i.e., when `spec.gateway.externalRgwEndpoints
 vhost-style addressing should be configured on the host cluster, and `hosting.dnsNames` is
 irrelevant. The default `advertiseEndpoint` for external CephObjectStores is the first entry in the
 `spec.gateway.externalRgwEndpoints` list, which users should be able to override if desired.
+
+### Advanced user configuration
+
+RGW is a very active project with many existing features and regular new feature additions. Rook
+maintainers have a hard time catching up with RGW's latest features. In an effort to help users who
+want access to rare and new features, Rook can allow users to set arbitrary Ceph configs in their
+CephObjectStore CR. RGW config reference: https://docs.ceph.com/en/reef/radosgw/config-ref/
+
+This is an advanced feature. It allows users to modify many untested aspects of the RGW and object
+store. Any user configs that break the object store should be a responsibility of the user.
+
+```yaml
+spec:
+  metadataPool: # ...
+  dataPool: # ...
+  gateway: # ...
+
+  cephConfig:
+    rgw_enable_apis: s3, s3website, swift, swift_auth, sts, iam, notifications # e.g. 'admin' removed
```

Comment:
With #15064 enabling the rgw apis has a new option in the CRD, so shall we remove this line of the example? Even though it would be possible here, we would only want to suggest using the other setting, right?

Reply:
This is actually on purpose. @arttor and I have the idea that we will remove that specific addition in #15064 when this feature is implemented, keeping the object API a bit less cluttered

Reply:
Ah, right. Enable APIs might require a restart. Options that require a restart seem like reasonable things to keep as specific options. We can discuss more in huddle.